runtime/j9vm/java11vmi.c

```diff
@@ -1305,6 +1305,7 @@ JVM_AddReadsModule(JNIEnv * env, jobject fromModule, jobject toModule)
 			/* Slightly different then check above since above I was dealing with the stack addr */
 			if (j9FromMod != j9ToMod) {
 				rc = allowReadAccessToModule(currentThread, j9FromMod, j9ToMod);
+				TRIGGER_J9HOOK_READS_MODULE_ADDED(((J9JavaVM *)vm)->hookInterface, currentThread, j9FromMod, j9ToMod);
```

Comment:
should only be triggered after sucessfull addition. So this should be done after checking `rc`